src/common/njord.ts

```diff
@@ -113,6 +113,7 @@ export type TransactionProps = {
   productId: string;
   receiverId: string;
   note?: string;
+  flags?: object;
```

Comment:
let's type it as `Partial<{ sourceId: string }>`

Reply:
you can also add it to UserTransaction flags and then `Pick<UserTransactionFlags>`